apps/bfd-server/dev/bfd-data-dictionary/README.md

```diff
@@ -0,0 +1,28 @@
+# BFD Data Dictionary
+
+The BFD Data Dictionary project provides data dictionary and mapping information for the Beneficiary FHIR Data Server.
+
+
+## Design
+
+The /data folder contains JSON respresentations of each element in the DD, separated into folders for major BFD version V1 and V2.
+
+The /app folder contains simple python scripts to read and transform content into external/user facing formats e.g. CSV, HTML, PDF, etc
+
+The python scripts are extremely crude/basic at the time of this writing.  
+It is likely they will be refactored/rewritten by someone more capable than me.
+(they need to be bulletproofed, use libraries like pandas, be ready for incorporation into CI processes, etc. etc.
+
+Runner.ps1 is a crude powershell script that demonstrates command line usage of each of the scripts 
```

Comment:
Maybe put this under "running locally"?

Reply:
I have removed the runner.ps Powershell script altogether and placed the relevant content under "Running Locally" as suggested